docs-2.0/nebula-dashboard-ent/4.cluster-operator/5.operation-record.md

```diff
@@ -0,0 +1,7 @@
+# Operation record
+
+This topic will describe the operation record in Nebula Dashboard.
+
+![operation record](../figs/ds-018.png)
+
+On the **Operation Record** page, you can check the operation record of the latest 1 hour, 6 hours, 1 day, 3 days, 7days or 14 days, including operator， operation， time and cluster name.
```

Comment:
record -->records

Reply:
[including operator， operation， time and cluster name.] sounds odd.

Reply:
```suggestion
On the **Operation record** page, you can check the operation record of the latest 1 hour, 6 hours, 1 day, 3 days, 7days, or 14 days. You can also view who runs what operation on which cluster at what time.
```